content/c-sharp/concepts/strings/terms/format/format.md

```diff
@@ -0,0 +1,237 @@
+| `f` to `fffffff` | The fraction of a second: tenth (`f`), hundredths (`ff`), ..., or ten millionths (`fffffff`).                             | `HH:mm:ss.fff` >> `20:05:09.000`   |
+| `F` to `FFFFFFF` | If non-zero, the fraction of a second: tenth (`F`), hundredths (`FF`), ..., or ten millionths (`FFFFFFF`).                | `HH:mm:ss.fFF` >> `20:05:09.0`     |
+| `t`, `tt`        | The first character of the AM/PM designator (`t`) or the entire one (`tt`).                                               | `hh:mm t` >> `08:05 P`             |
+| `\`              | Escape character to display characters otherwise interpreted as specifiers. It must be doubled in quoted string literals. | `\da\y: d` >> `day: 7`             |
+| any other        | Displayed as is.                                                                                                          | `MM-dd-yyyy` >> `06-07-2009`       |
+
+For example, `MMM dd, yyyy` applied to `new DateTime(2024, 6, 1)` returns `Jun 01, 2024`.
+
+### Enum Formats
+
+There are only standard formats available to format enum values. Some of these are:
+
+- `G`: An enum value is displayed as its enum name. A binary OR (`\|`) combination of enum values ​​is displayed as a comma-separated list of matching names. An integer (cast to an enum type) is displayed as the name of the matching enum value if found. Otherwise, an attempt is made to find a list of matching names, but only with the enum type defined with the `Flags` attribute. If no match is found, the integer is displayed.
+- `F`: Like `G`, but attempts to display a list of names even if the enum type is not defined with the `Flags`attribute.
+- `D`: Displays the integer corresponding to the enum value.
+
+## Example
+
+The following example shows how to compose a message using the `.Format()` method and how the result is affected by the culture involved in the formatting procedure. The message is generated twice: once with the current culture (in this case, `en-US`) and once with the British culture (`en-GB`). The example also shows how to use the same parameter specifier in different format items to format the same value (`loginDate`), once as a date and once as a time.
+
+```cs
+using System;
+using System.Globalization;
+
+public class Example {
+
+  public static void Main (string[] args)
+ {
+    string name = "John";
+    DateTime loginDate = new DateTime(2024, 9, 2, 17, 45, 0);
+    string formatString = "Hi {0}, your last login was on {1:d} at {1:t}.";
+
+ // Displaying a message using the current culture:
+
+    string message = String.Format(formatString, name, loginDate);
+    Console.WriteLine(message);          
+
+ // Displaying the same message but using an Italian culture:
+
+    CultureInfo culture = CultureInfo.CreateSpecificCulture("en-GB");
+    message = String.Format(culture, formatString, name, loginDate);
+
+    Console.WriteLine(message);
+ }
+}
+```
```

Comment:
This example section should be near the syntax section so that it's easy for the reader to study the syntax and then see how it works in the example, and then see what more customizations can be done using different format specifiers.

Also, this example should be broken down into two different examples - the first example simply shows how `.Format()` works, and then as the next step, we can show how different cultures can be specified as well.